### PR TITLE
Add SOXS wordmark and update README

### PR DESCRIPTION
**Pull request by granttremblay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
-[![astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat)](http://www.astropy.org/)  
-[![Build Status](https://travis-ci.org/XRStools/soxs.svg?branch=master)](https://travis-ci.org/XRStools/soxs)  
-[![Coverage Status](https://coveralls.io/repos/github/XRStools/soxs/badge.svg?branch=master)](https://coveralls.io/github/XRStools/soxs?branch=master)  
+
+
+![SOXS: Simulated X-ray Observations of Astrophysical Sources](misc/SOXS_Wordmark.png)
```
Comment (jzuhone) [inline README.md]: This is "Simulated Observations of X-ray Sources"

File: README.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+
+[![astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat)](http://www.astropy.org/)  [![Build Status](https://travis-ci.org/XRStools/soxs.svg?branch=master)](https://travis-ci.org/XRStools/soxs)  [![Coverage Status](https://coveralls.io/repos/github/XRStools/soxs/badge.svg?branch=master)](https://coveralls.io/github/XRStools/soxs?branch=master)  
 
 # What is SOXS?
 
 SOXS is a software suite which creates simulated X-ray observations of
-astrophysical sources with the mission concept X-ray Surveyor. The goal of
+astrophysical sources with almost *any* existing or planned X-ray observatory. The goal of
```
Comment (jzuhone) [inline README.md:10]: Maybe say "can create".

File: README.md (hunk 1 above)
Comment (granttremblay) [reply README.md]: whoops! Fixed

File: README.md (hunk 2 above)
Comment (jzuhone) [reply README.md:10]: @granttremblay did you see this one?

**Review by jzuhone — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (granttremblay) [reply README.md:10]: yup, fixed!

**Review by jzuhone — APPROVED** [review]

**Pull request merged**